common.go

```diff
@@ -51,10 +51,14 @@ type ResStringPoolHeader struct {
 	StylesStart uint32
 }
 
+type ResStringPoolSpan struct {
+	FirstChar, LastChar uint32
+}
+
 type ResStringPool struct {
 	Header  ResStringPoolHeader
 	Strings []string
-	Styles  []string
+	Styles  []ResStringPoolSpan
```

Comment:
I was misunderstood about Styles.
They are not string, ResStringPool_span.

https://android.googlesource.com/platform/frameworks/base/+/master/libs/androidfw/ResourceTypes.cpp#844